src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -133,12 +133,16 @@ public IHttpActionResult GetHostStatus()
             }
         }
 
+        [HttpGet]
         [HttpPost]
         [Route("admin/host/ping")]
         [AllowAnonymous]
-        public IHttpActionResult Ping()
+        public HttpResponseMessage Ping()
         {
-            return Ok();
+            var response = new HttpResponseMessage(HttpStatusCode.OK);
+            response.Headers.Add("Cache-Control", "no-store, no-cache");
```

Comment:
Returns same header values as in the dev branch ASP.NET Core version. ASP.NET WebAPI doesn't have a declarative cache control attribute.